contracts/potlock/src/potlock_admin_interactions.rs

```diff
@@ -0,0 +1,111 @@
+use crate::potlock_storage::{self, PotlockId, ProjectId, Status};
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+pub type ProjectPercentage = MultiValue2<usize, u64>;
+pub const MAX_PERCENTAGE: u64 = 10_000; // 100%
+
+#[multiversx_sc::module]
+pub trait PotlockAdminInteractions:
+    potlock_storage::PotlockStorage + multiversx_sc_modules::only_admin::OnlyAdminModule
+{
+    #[only_admin]
+    #[endpoint(changeFeeForPots)]
+    fn change_fee_for_pots(&self, token_identifier: TokenIdentifier, fee: BigUint) {
+        require!(
+            token_identifier.is_valid_esdt_identifier(),
+            "Invalid token provided"
+        );
+        require!(
+            token_identifier.is_valid_esdt_identifier() && fee.ge(&BigUint::zero()),
```

Comment:
Also, here the fee shouldn't be strictly greater than 0? Right now it checks if it is greater or equal. For better clarity, I would keep the checks in separate lines. So one for valid esdt and the other one for >0 check.

Reply:
Btw, simply use the `>` or `>=` operator instead, and just use `0` instead of `BigUint::zero()`, way easier to read that way.

And yeah, agreed with Sorin, keep them in different requires.